docs/content/users/install/ddev-installation.md

```diff
@@ -114,9 +114,38 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
     * Installing Chocolatey package manager (optional).
     * One time initialization of mkcert.
     * Installing WSL2 and installing a distro like Ubuntu.
-    * Installing or upgrading to the latest Docker Desktop for Windows with WSL2 enabled.
+    * (Optionally) Installing Docker Desktop for Windows and enabling WSL2 integration with the distro.
```

Comment:
Move “optional” to the end in the style of the first bullet:

```suggestion
    * Installing Docker Desktop for Windows and enabling WSL2 integration with the distro (optional).
```

Reply:
```suggestion
    * Installing Docker Desktop for Windows and enabling WSL2 integration with the distro (optional, do this if you're using the Docker Desktop approach).
```